release-notes/opensearch-knn.release-notes-2.10.0.0.md

```diff
@@ -2,8 +2,6 @@
 
 Compatible with OpenSearch 2.10.0
 
-### Features
-* Add Clear Cache API ([#740](https://github.com/opensearch-project/k-NN/pull/740))
```

Comment:
I think we need to strike through so we leave a trace for it:
* ~Add Clear Cache API ([#740](https://github.com/opensearch-project/k-NN/pull/740))~ Feature was mistakenly added to the release notes, although it was not included in the release.